easybuild/easyconfigs/n/NGSadmix/NGSadmix-32-GCC-7.3.0-2.30.eb

```diff
@@ -0,0 +1,36 @@
+# Contribution from the Crick HPC team
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CmdCp'
+
+name = 'NGSadmix'
+version = '32'
+CXX = 'g++'
+CXXFLAGS = '-O3'
```

Comment:
@sassy-crick These two lines should be removed.

They have no effect at all (you're defining local variables in the easyconfig file here, not environment variables), and the toolchain mechanism will define `$CXX` and `$CXXFLAGS` during the `prepare` step of the installation procedure.

Reply:
@vanzod thanks! I checked and it was picking up the OS zlib and hence I never spotted it.
@boegel thanks! Again the help is much appreciated.